__tests__/app.test.js

```diff
@@ -26,4 +23,18 @@ describe('/api/topics', () => {
             expect(topic).toHaveProperty("description", expect.any(String))
         })
     })
+})
+
+describe('/api', () => {
+    test('should return an object with all the available endpoints of the api', async () => {
+        const { body: { endpoints } } = await request(app)
+            .get('/api')
+            .expect(200)
+        expect(endpoints).toBeInstanceOf(Object)
+        expect(endpoints).toHaveProperty("GET /api")
+        expect(endpoints).toHaveProperty("GET /api/topics")
+        Object.values(endpoints).forEach(value => {
+            expect(value).toHaveProperty('description', expect.any(String))
+        })
+    })
 })
```

Comment:
I would also add a test to check that the response looks like the ```endpoints.json``` file. This will be dynamic and the test won't need to be updated as you add more endpoints to the json file